Assets/Plugins/Source/Editor/EpicOnlineServicesConfigEditor.cs

```diff
@@ -70,20 +53,15 @@ public static void AddPlatformSpecificConfigEditor(IPlatformSpecificConfigEditor
         [MenuItem("Tools/EOS Plugin/Dev Portal Configuration")]
         public static void ShowWindow()
         {
-            GetWindow(typeof(EpicOnlineServicesConfigEditor), false, "EOS Config Editor", true);
+            GetWindow<EpicOnlineServicesConfigEditor>("EOS Config Editor");
```

Comment:
In all cases where we utilize the `GetWindow` function, calling the templated method as done here provides identical behavior in a manner that is much more clear.